src/Gitlab/Client.ts

```diff
@@ -134,6 +134,17 @@ export class GitLabClient {
         }
     }
 
+    public async getProjectAccessLevel(id: ProjectId): Promise<AccessLevel> {
+        try {
+            const me = await this.user();
+            const { data } = await axios.get(`api/v4/projects/${encodeURIComponent(id)}/members/${me.id}`, this.defaultConfig);
+            return data.access_level as AccessLevel;
```

Comment:
I wish there was some type and value validation of access_level.
That way it would be easier to spot if the API breaks.

Reply:
Added type checking, though we probably need to figure out how to do this as a whole.